akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -685,6 +702,13 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
         }
       }
 
+      // FIXME remove
+      if (latestGossip.version.versions.size > latestGossip.members.size) {
```

Comment:
I only used this for sanity check when running tests, it should be removed as the FIXME says